_episodes/20-carpentries.md

```diff
@@ -56,71 +56,15 @@ community structure, and policies in January, 2018. Each organization will maint
 
 ## The Carpentry Community
 
-The Carpentry Community has several parts, which function together to spread data skills and 
-computational literacy among researchers and other professionals worldwide. 
-
-- **Instructors (Mentors)** teach workshops and form the cornerstone of the Carpentry community. They help new instructors integrate into the community by leading discussion sessions. Some also lead small [mentoring groups](https://software-carpentry.org/join/subcom/mentoring/) to help new instructors prepare for their first few workshops.
-
-- **Trainers** are Carpentry instructors who have received additional training in pedagogy. They train new instructors, develop the instructor training curriculum, and lead teaching demonstrations.
-
-- **Lesson Developers** create Carpentries lessons for new subject areas.
-
-- **[Lesson Maintainers](https://software-carpentry.org/join/subcom/maintainers/)** oversee curricular changes and maintain the high-quality of Carpentry lessons.
-
-- **Lesson Infrastructure Subcommittee** members make decisions about and implement changes to lesson templates to ensure a good user experience across our lessons.
-
-- **Assessment Subcommittee** members help us to understand the impact of our workshops on the learners who attend and guide the development, implementation, and instruction of our workshops based on these assessments.
-
-- The **Executive Council** makes strategic decisions about the future of the organization and its operations.
-
-- **Champions** advocate for the Carpentries within their home institution and region, growing a local network of Carpentry instructors and supporting workshop learners.
-
-- **Infrastructure Team** helps with AMY (or tracking system), data analysis, surveys, and providing systems administration and other support infrastructure functions.
-
-- **Workshop Coordinators** help to plan and run Carpentries workshops at their institutions.
+The Carpentries works to help institutions and individuals spread skills for data analysis, computational thinking, and research software development through building local and global communities of practice. Our community depends on individuals like you who are passionate about expanding these communities of practice through inclusive and evidence-based instructional practices, and can contribute your perspective and expertise to continually refine our instructional materials and practices.  A full description of the bread and diversity of roles that members play is beyond the scope of this new instructor training. Detailed descriptions of community member roles, an overview of the Carpentries' various social media channels, a calendar of future community events, and descriptions of mailing lists used by the community can be found [here on the Carpentries website](http://static.carpentries.org/community/).
```

Comment:
"bread and diversity" --> "breadth and diversity"

I would also take out the sentence "A full description of the bread and diversity of roles that members play is beyond the scope of this new instructor training" because we *do* discuss it in Instructor Training, the information is just on a different page.  So I would combine that sentence + the one following to say: 
"A full description of the breadth and diversity of community member roles, an overview of the Carpentries various social media channels..." etc.